docs/configuration.md

```diff
@@ -1102,6 +1102,27 @@ Apart from these, the following properties are also available, and may be useful
   </td>
   <td>3.0.0</td>
 </tr>
+<tr>
+  <td><code>spark.shuffle.service.db.enabled</code></td>
+  <td>true</td>
+  <td>
+    To determine whether external shuffle uses db to store shuffle state.
+    In standalone and Yarn modes, for more detail, see
+    <a href="spark-standalone.html#Cluster Launch Scripts">this description</a>.
+  </td>
+  <td>3.0.0</td>
+</tr>
+<tr>
+  <td><code>spark.shuffle.service.db.backend</code></td>
+  <td>LEVELDB</td>
+  <td>
+    To specify a disk-based store used in shuffle service local db when 
+    <code>spark.shuffle.service.db.enabled</code> is true, LEVELDB or ROCKSDB.
+    In standalone and Yarn modes, for more detail, see
+    <a href="spark-standalone.html#Cluster Launch Scripts">this description</a>.
```

Comment:
ditto

Reply:
Is it appropriate to add `spark.shuffle.service.db.backend` to `running-on-yarn.md`? The configurations listed in `running-on-yarn.md` are all in `spark.yarn.shuffle` namespace now.

Reply:
If appropriate, I will add `spark.shuffle.service.db.backend`  to `running-on-yarn.md` and link it

Reply:
is [0920f0e](https://github.com/apache/spark/pull/37610/commits/0920f0e000e638f15d283e750b639d41b3501fab) ok ?